### PR TITLE
[#722] test: cleanup residue files in tmp directory after tests

### PR DESCRIPTION
**Pull request by cchung100m**
Hi @kaijchen @zuston @jerqi @xianjingfeng 

Any comments and suggestions would be appreciated if you are available, thank you.

### What changes were proposed in this pull request?

Cleanup residue files in tmp directory after tests

### Why are the changes needed?

Fix: #722 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

current UT

### CODE REVIEW

File: common/src/test/java/org/apache/uniffle/common/util/ChecksumUtilsTest.java [hunk 1]
```diff
@@ -21,18 +21,27 @@
...
 
 public class ChecksumUtilsTest {
 
+  @TempDir File tempDir;
+
+  @BeforeEach
+  void beforeEach() {
+    assertTrue(this.tempDir.isDirectory());
+  }
+
```
Comment (kaijchen) [inline common/src/test/java/org/apache/uniffle/common/util/ChecksumUtilsTest.java:44]: This beforeEach is not necessary.

Comment (cchung100m) [reply common/src/test/java/org/apache/uniffle/common/util/ChecksumUtilsTest.java]: Thank you for the feedback and I updated the part you mentioned.

**Review by zuston — APPROVED** [review]
lgtm. cc @kaijchen to take a look again

**Review by kaijchen — APPROVED** [review]
LGTM, thanks @cchung100m for the fix.

**Pull request merged**